src/workerd/api/streams/standard.c++

```diff
@@ -2968,124 +2971,139 @@ private:
         // be freed. When the JS promise resolves, we make sure we detect that
         // case and handle appropriately (generally by canceling the readable
         // and exiting the loop).
+        return js.tryCatch([&] {
```

Comment:
The diff here is unfortunate... the change is smaller than it looks here. Basically just wrapped everything in the js.tryCatch.

Reply:
Fix by hiding whitespace:

![Screenshot from 2023-05-17 09-27-25](https://github.com/cloudflare/workerd/assets/4001805/e2c170c9-24a3-4dd4-9966-dbf9033df734)